### ISSUE TITLE
`npm install` on a fresh clone is ending with a purge

### ISSUE THREAD

**Issue by r-park** [angular/angular#5172]
`npm install` on a fresh clone is ending with a purge. Followed [these steps](https://github.com/angular/angular/blob/master/DEVELOPER.md) – wondering what am I missing?
```
:-( npm dependencies are stale or in an in unknown state!
    purging...
```
- OSX Yosemite
- node v5.0.0
- npm  v3.3.9

**Comment by gkalpak** [comment]
This is because of the changes in npm `v3` (which comes with node `v5`).
There is the same issue with ng1 :( (See https://github.com/angular/angular.js/issues/13123.)

**Comment by r-park** [comment]
@gkalpak thanks for the info

**Comment by gkalpak** [comment]
I didn't mean to imply this should be closed, btw.
I think this is an issue that should be addressed.

**Comment by pkozlowski-opensource** [comment]
I believe that this is fixed in master via 05266241af9973c33cf7ab44a8ca00aeca33ba0d
